include/vigra/mathutil.hxx

```diff
@@ -164,6 +164,17 @@ VIGRA_DEFINE_MISSING_ABS(signed long long)
 using std::isinf;
 using std::isnan;
 
+#ifdef VIGRA_NO_INT_ISNAN
```

Comment:
I just realize that the functions `contains_nan` and `contains_inf` need polishing anyway, as they use the potentially slow `in[ii]` (with `ii` being interger) to access the data.